From 132275dca040821cf44e511f31c6b737d862161e Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard@netgear.com>
Date: Tue, 10 Nov 2015 11:54:53 -0800
Subject: [PATCH 33/33] phy: marvell: leave default LED settings

By default, the driver writes in some default LED register values for
88E1318, which is used in our Marvell-based platforms.  These default
values are different from what U-Boot sets, and different from what
they were in ReadyNASOS 6.2.x.  So stop setting these values, and just
use whatever U-Boot set up.
---
 drivers/net/phy/marvell.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/net/phy/marvell.c b/drivers/net/phy/marvell.c
index 0240552..e177b5a 100644
--- a/drivers/net/phy/marvell.c
+++ b/drivers/net/phy/marvell.c
@@ -385,12 +385,6 @@ static int m88e1121_config_aneg(struct phy_device *phydev)
 	if (err < 0)
 		return err;
 
-	oldpage = phy_read(phydev, MII_MARVELL_PHY_PAGE);
-
-	phy_write(phydev, MII_MARVELL_PHY_PAGE, MII_88E1121_PHY_LED_PAGE);
-	phy_write(phydev, MII_88E1121_PHY_LED_CTRL, MII_88E1121_PHY_LED_DEF);
-	phy_write(phydev, MII_MARVELL_PHY_PAGE, oldpage);
-
 	err = genphy_config_aneg(phydev);
 
 	return err;
-- 
1.9.1

